advanced/mathfx/src/main/java/org/arakhne/afc/math/geometry/d2/ifx/MultiShape2ifx.java

```diff
@@ -230,7 +230,16 @@ public void invalidated(Observable observable) {
 				fireChange(new SimpleUpdateChange<>(position, this));
 			}
 		}
-
+		public boolean equals(Object obj) {
+			if (! super.equals(obj)) {
+				return false;
+			}
+			InternalObservableList lObj = (InternalObservableList) obj;
+			if (internalList.equals(lObj.internalList)) {
+				return true;
+			}
+			return false;
+		}
 	}
 
```

Comment:
The implementation of the `hashCode()` is missed. When a `equals(Object)` function is coded, the `hashCode()` function must be too.